src/test/java/org/tarantool/MsgPackLiteSerializeTest.java

```diff
@@ -0,0 +1,602 @@
+package org.tarantool;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.tarantool.TestAssertions.assertSubArrayEquals;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.math.BigInteger;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Test set for {@link MsgPackLite#pack(Object, OutputStream)} method.
+ */
+public class MsgPackLiteSerializeTest {
+
+    private MsgPackLite msgPackLite;
+    private ByteArrayOutputStream outputStream;
+
+    @BeforeEach
+    void setUp() {
+        msgPackLite = new MsgPackLite();
+        outputStream = new ByteArrayOutputStream();
+    }
+
+    /**
+     * Nil format stores nil in 1 byte.
+     *
+     * nil:
+     * +--------+
+     * |  0xc0  |
+     * +--------+
+     *
+     * @throws IOException if any IO errors occur
+     */
+    @Test
+    void testPackNull() throws IOException {
+        byte[] expectedOutput = { (byte) 0xc0 };
+
+        msgPackLite.pack(null, outputStream);
+        assertArrayEquals(expectedOutput, outputStream.toByteArray());
+    }
+
+    /**
+     * Bool format family stores false or true in 1 byte.
+     *
+     * false:
+     * +--------+
+     * |  0xc2  |
+     * +--------+
+     *
+     * true:
+     * +--------+
+     * |  0xc3  |
+     * +--------+
+     */
+    @Test
+    void testPackBoolean() throws IOException {
+        byte[] expectedOutput = { (byte) 0xc2, (byte) 0xc3 };
+
+        msgPackLite.pack(false, outputStream);
+        msgPackLite.pack(true, outputStream);
+        assertArrayEquals(expectedOutput, outputStream.toByteArray());
+        outputStream.reset();
+
+        msgPackLite.pack(Boolean.FALSE, outputStream);
+        msgPackLite.pack(Boolean.TRUE, outputStream);
+        assertArrayEquals(expectedOutput, outputStream.toByteArray());
+        outputStream.reset();
+    }
+
+    /**
+     * positive fixnum stores 7-bit positive integer.
+     *
+     * +--------+
+     * |0XXXXXXX|
+     * +--------+
+     *
+     * 0XXXXXXX is 8-bit unsigned integer [0..127]
+     */
+    @Test
+    void testPackFixedPositiveInteger() throws IOException {
+        msgPackLite.pack(0, outputStream);
```

Comment:
We may verify byte / short / long, their boxed representations and BigInteger. It worth to add at least `byte` and `Integer` I think.